### ISSUE TITLE
[A11Y] Color pallet doesn't have enough contrast

### ISSUE THREAD

**Issue by Steve0Greatness** [lopste/forumoji#73]
Accent color to text color: [test results](https://color.a11y.com/ContrastPair/?bgcolor=ff8c00&fgcolor=ffffff), [(other way around)](https://color.a11y.com/ContrastPair/?bgcolor=ffffff&fgcolor=ff8c00)

Orange and white do not work well as accessible colors when together. A better color match for all use case of orange and black.

However, I don't think this would work well for the category toolbar nor as the background color of the emojis once selected; I'd suggest using black in-place of orange in these cases.

Here's what the page would look like if such changes were to be implemented:

![](https://github.com/lopste/forumoji/assets/75220768/69951ae6-d75b-4393-acf3-c6cbc7e6773a)

**Comment by lopste** [comment]
fixed in #72

**Comment by Steve0Greatness** [comment]
Hey there! That commit intentionally didn't address this issue.

**Comment by Steve0Greatness** [comment]
@lopste This wasn't fixed in #72. #72 did not address the color pallet of the website.

**Comment by lopste** [comment]
my bad sorry
